src/lapack/gpu/larft.cu

```diff
@@ -8,6 +8,9 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+#include <complex>
```

Comment:
Looks like we are missing `std::complex` from `inspect.toml`.

Reply:
Excellent point. I'm fixing those in #1243. There's a long list of includes missing...